src/Analyzers/CSharp/Tests/UseCollectionExpression/UseCollectionExpressionForArrayTests.cs

```diff
@@ -5348,4 +5348,355 @@ private void Test1(dynamic obj, int?[] args)
             LanguageVersion = LanguageVersion.CSharp12,
         }.RunAsync();
     }
+
+    [Fact]
+    public async Task PreservesTrailingTriviaAfterInitializer1()
+    {
+        await new VerifyCS.Test
+        {
+            TestCode = """
+                class C
+                {
+                    int[] i = [|[|new|][]|] //Test
+                    { 1, 2, 3 };
+                }
+                """,
+            FixedCode = """
+                class C
+                {
+                    int[] i =
+                    //Test
```

Comment:
This extra newline is caused by the elastic marker here: https://github.com/dotnet/roslyn/blob/540092f62346918037f444927c8cdef038519157/src/Analyzers/CSharp/Analyzers/UseCollectionExpression/UseCollectionExpressionHelpers.cs#L729-L733

This does a lot of heavy lifting in many other tests. Removing it would mean manually cleaning up a lot of different cases that are not currently catered for.